core/src/main/java/com/yahoo/oak/HashChunk.java

```diff
@@ -332,6 +332,11 @@ HashChunkIter chunkIter(ThreadContext ctx) {
         }
 
 
+        /**
+         * find the index of the next occupied entry in the entryies array
```

Comment:
The explanation should say the "first" index, this is what name of the function suggests...